libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.scss

```diff
@@ -1 +1,60 @@
 @import '~fundamental-styles/dist/fixed-card-layout';
+
+/** TODO: Move to styles */
```

Comment:
@platon-rov can you open a PR for this todo?

Reply:
Opened https://github.com/SAP/fundamental-styles/pull/3208

Reply:
Merged, this pr can be reviewed.